### PR TITLE
Update tutorial links and location

### PR DESCRIPTION
**Pull request by ggabernet**
Updated `contributor tutorial` location to developer tutorials, and fixed some broken links.

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1102"></a>

### CODE REVIEW

File: markdown/developers/developer_tutorials.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ subtitle: In-depth step-by-step guides for writing nf-core pipelines
 
 This page includes step-by-step guides, walkthroughs, and general long-form introductions to various aspects of nf-core.
 
+- [Tutorial: nf-core contributor overview](tutorials/nf_core_contributing_overview.md)
+  - An overview on contributing to nf-core, starting with installing nf-core tools, creating a new pipeline using the nf-core template, adding modules to it and releasing.
```
Comment (christopher-hakkaart) [inline markdown/developers/developer_tutorials.md:11]:
```suggestion
  - An overview of how to contribute to nf-core, starting with installing nf-core tools, creating a new pipeline using the nf-core template, adding modules to a pipeline, and releasing a pipeline.
```

Comment (christopher-hakkaart) [reply markdown/developers/developer_tutorials.md:11]: If you don't like the changes feel free to ignore.

**Review by christopher-hakkaart — APPROVED** [review]
@ggabernet Looks good. I left one wording suggestion that I think would help improve meaning but otherwise no problems.

File: markdown/developers/developer_tutorials.md (hunk 1 above)
Comment (ggabernet) [reply markdown/developers/developer_tutorials.md]: Thanks for the input @christopher-hakkaart

**Comment by ggabernet** [comment]
Thanks, the link checking GitHub actions is playing crazy at the moment so I'm gonna merge anyways...

**Pull request merged**